tests/queries/0_stateless/02473_optimize_old_parts.sh

```diff
@@ -65,3 +66,32 @@ SELECT sleepEachRow(1) FROM numbers(9) SETTINGS function_sleep_max_microseconds_
 SELECT (now() - modification_time) > 5 FROM system.parts WHERE database = currentDatabase() AND table='test_with_merge' AND active;
 
 DROP TABLE test_with_merge;"
+
+# Partition 2 will ignore max_bytes_to_merge_at_max_space_in_pool
```

Comment:
Resolve conflict and I would move both test additions into a new test.
Create new files for both tests and add your changes.